pkg/apply/scale.go

```diff
@@ -54,6 +54,23 @@ func NewScaleApplierFromArgs(scaleArgs *ScaleArgs, flag string) (applydrivers.In
 	if scaleArgs.Cluster.Nodes == "" && scaleArgs.Cluster.Masters == "" {
 		return nil, fmt.Errorf("the node or master parameter was not committed")
 	}
+	if scaleArgs.fs != nil {
+		if scaleArgs.fs.Changed("user") || cluster.Spec.SSH.User == "" {
+			cluster.Spec.SSH.User = scaleArgs.SSH.User
+		}
+		if scaleArgs.fs.Changed("pk") || cluster.Spec.SSH.Pk == "" {
+			cluster.Spec.SSH.Pk = scaleArgs.SSH.Pk
+		}
+		if scaleArgs.fs.Changed("pk-passwd") || cluster.Spec.SSH.PkPasswd == "" {
+			cluster.Spec.SSH.PkPasswd = scaleArgs.SSH.PkPassword
+		}
+		if scaleArgs.fs.Changed("port") || cluster.Spec.SSH.Port == 0 {
+			cluster.Spec.SSH.Port = scaleArgs.SSH.Port
+		}
+		if scaleArgs.fs.Changed("passwd") || cluster.Spec.SSH.Passwd == "" {
+			cluster.Spec.SSH.Passwd = scaleArgs.SSH.Password
+		}
+	}
```

Comment:
Currently, we may do not support having multiple passwords/pk in a cluster. If you replace `cluster.Spec.SSH` here, it may result in the modification of the original password and the inability of the original nodes to connect normally.

Reply:
> Currently, we may do not support having multiple passwords/pk in a cluster.

actually, we did. override the `.spec.hosts[*].ssh` field.